examples/OpticalTracker/README.md

```diff
@@ -0,0 +1,47 @@
+Proximity Focusing RICH
+=======================
+
+Example RICH demonstrating `Geant4OpticalTracker` Sensitive Detector plugin usage
+
+![PFRICH](doc/geometry.png)
+
+For testing, it is recommended to follow `../README.md`. See below for guidance for
+running this example standalone.
+
+## Local Development
+The following assumes that your current working directory is `DD4hep/examples/OpticalTracker`.
+
+Build with `cmake`, for example:
+```bash
+cmake -B build -S . -D CMAKE_INSTALL_PREFIX=install
+cmake --build build -- install
+```
+
+View the geometry (using `dd_web_display`, currently a tool from [NPDet](https://eicweb.phy.anl.gov/EIC/NPDet));
```

Comment:
I'd leave out the dd_web_display stuff. This would be geoDisplay in dd4hep. We just don't use it because it isn't always reliable in a container.

Reply:
Removed this and `view.sh`.